resources/elk5-old_compatibility/docker/1000_nessus_process_file.conf

```diff
@@ -6,25 +6,13 @@
 
 
 input {
-  file {
-    path => "/opt/VulnWhisperer/data/nessus/**/*.csv"

```

Comment:
We will need to remember when we release version 2.0, for those users who are using ELK, that there will be no backwards compatibility with the Nessus CSV output, although that shouldn't be an issue as everything downloaded in CSV should have already parsed.
I totally agree on deleting the csv part and simplifying the project.